README.md

```diff
@@ -36,7 +36,7 @@ In a browser:
 
 Using npm:
 ```shell
-$ npm i -g npm
+$ npm i -g lodash
```

Comment:
This is not a typo, it is the correct way to install the latest npm

https://docs.npmjs.com/downloading-and-installing-node-js-and-npm